### PR TITLE
data:AyebakuroOruwori

### PR DESCRIPTION
**Pull request by AyebakuroOruwori**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5109"></a>

### CODE REVIEW

File: data/AyebakuroOruwori.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "icon": "FaInstagram"
+    }, 
+
+  {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ayebakuro-oruwori/",
+      "icon": "FaLinkedIn"
+    },
+
+  {
```
Comment (amandamartin-dev) [inline data/AyebakuroOruwori.json:25]: You added Github twice, otherwise looks good!

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: data/AyebakuroOruwori.json (hunk 1 above)
Comment (AyebakuroOruwori) [reply data/AyebakuroOruwori.json:25]: Thank you, I'll update it right away

**Review by amandamartin-dev — DISMISSED** [review]

**Review by SurajPratap10 — APPROVED** [review]

File: data/AyebakuroOruwori.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "avatar": "https://github.com/AyebakuroOruwori/hashNodeBlog/blob/main/66691177.jpeg",
```
Comment (Panquesito7) [inline data/AyebakuroOruwori.json:2]: The `avatar` field is no longer required as of a recent update.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/AyebakuroOruwori.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  {
+      "name": "GitHub",
+      "url": "https://github.com/AyebakuroOruwori",
+      "icon": "FaGithub"
+    },
+
+  {
+      "name": "Hashnode",
+      "url": "https://ayebakuro.hashnode.dev/",
+      "icon": "FaHashnode"
```
Comment (Panquesito7) [inline data/AyebakuroOruwori.json:34]: Wrong icon name.
```suggestion
      "icon": "SiHashnode"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/AyebakuroOruwori.json (hunk 3 above)
Comment (AyebakuroOruwori) [reply data/AyebakuroOruwori.json]: Thank You

File: data/AyebakuroOruwori.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+   {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/ayebakuro_o/",
+      "icon": "FaInstagram"
+    }, 
+
+  {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ayebakuro-oruwori/",
+      "icon": "FaLinkedIn"
```
Comment (Panquesito7) [inline data/AyebakuroOruwori.json:22]: Wrong icon name.
```suggestion
      "icon": "FaLinkedin"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SurajPratap10 — APPROVED** [review]
LGTM

I just have a doubt regarding the type as everyone writes `public` there else everything is fine.

**Comment by AyebakuroOruwori** [comment]
> 

What do you suggest ?

**Comment by AyebakuroOruwori** [comment]
> 

What do you suggest ?

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Comment by kumarsonsoff3** [comment]
Hey @AyebakuroOruwori, not sure why did you close your PR after it is being approved by 2 maintainers. It should be ready to merge!?

**Review by amandamartin-dev — APPROVED** [review]
lgtm!

**Pull request merged**